### PR TITLE
nx-read-only grid - RSC-545

### PR DESCRIPTION
**Pull request by rpokorny**
https://issues.sonatype.org/browse/RSC-545

### CODE REVIEW

**Comment by glenhunter** [comment]
Why do we need both the grid example on `nx-read-only` and the Layout Example page? It's confusingly redundant.

File: gallery/src/styles/NxReadOnly/NxReadOnlyPage.tsx [hunk 1]
```diff
@@ -58,14 +76,37 @@ const NxReadOnlyPage = () => {
...
       </GalleryExampleTile>
+
+      <GalleryExampleTile title="Grid Example"
+                          id="nx-read-only-grid-example"
+                          codeExamples={gridCode}
+                          htmlExample={nxReadOnlyGridExampleCode}>
+        An <NxCode>nx-read-only</NxCode> arranged in a grid. The exact layout of the grid is custom to this example.
+        In typical usage, the layout of the grid would be usage-specific requiring the use
+        of <NxCode>grid-template</NxCode> and/or <NxCode>grid-area</NxCode> CSS properties.
+      </GalleryExampleTile>
```
Comment (glenhunter) [inline gallery/src/styles/NxReadOnly/NxReadOnlyPage.tsx:109]: I think that it's worth calling out the use of `nx-read-only__item` here as well. 

e.g. "Note the use of `nx-read-only__item` wrapping each set of `dt/dd`s."

**Comment by glenhunter** [comment]
If we keep both examples I think that we should call out the existence of the layout page on both the `nx-grid` and `nx-read-only` pages.

**Comment by rpokorny** [comment]
> Why do we need both the grid example on nx-read-only and the Layout Example page? It's confusingly redundant. I understand why you have it, I think that it could just be incorporated into the nx-read-only page.

It's similar to why we have examples on the `NxFormGroup` page and then also the "Form Layout Styles" page.  One to show the component in isolation and another to show how several components can be used together to implement a larger common pattern.

**Comment by rpokorny** [comment]
> If we keep both examples I think that we should call out the existence of the layout page on both the nx-grid and nx-read-only pages.

I added a link to it on the `nx-read-only` page in 55e444e5.  I don't feel like it makes as much sense to link to it from the `nx-grid` page as well.

File: gallery/src/styles/NxReadOnly/NxReadOnlyPage.tsx (hunk 1 above)
Comment (rpokorny) [reply gallery/src/styles/NxReadOnly/NxReadOnlyPage.tsx:116]: > I think that it's worth calling out the use of nx-read-only__item here as well.

97c9a682

**Review by glenhunter — APPROVED** [review]

**Pull request merged**